src/features/pastelStatistics/__tests__/pastelLib.test.ts

```diff
@@ -45,6 +45,15 @@ describe('managePastelDatabase', () => {
       1621518133357,
     ],
   ]
+  const mockNetTotals = [
```

Comment:
I think this mock belong to the NetTotals chart, right?

Reply:
This is the nettotals table's mock data.

Reply:
Ok So please remove it because it is not related to this PR.